x/pubkey/types/errors.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import "cosmossdk.io/errors"
+
+var (
+	ErrEmptyValue        = errors.Register("pubkey", 1, "empty value")
+	ErrInvalidType       = errors.Register("pubkey", 2, "invalid type")
+	ErrValidatorNotFound = errors.Register("pubkey", 3, "validator not found")
```

Comment:
Do these custom errors add value over the SDK default errors or can we simply wrap a standard error with a description?